### ISSUE TITLE
Add a Button with a link

### ISSUE THREAD

**Issue by elpopi2003** [h5p/h5p-memory-game#21]
Hi!

It would be awesome if you guys could insert a button that shows once you complete the game, redirects to some URL when you click it...

By the way, no "Try again" button shows once you finish a game, and I've got that checked in the config.

Thanks!!

**Comment by icc** [comment]
I'm sorry about the _Try again_, it looks like functionality was missing from the released version – it should start working in the upcoming version.

Are you thinking of a button that leads to another task or something like that?

**Comment by stephensiano** [comment]
We're setting up a trade show game using Drupal. We'd like to randomize a _Image Pairing_ question with a view - no worries - but once the single Image Pairing question is finished, we'd like to use that BUTTON -- instead of a retry -- we'd like to use the Button go to another Drupal Node. Is that something I can request here?

**Comment by icc** [comment]
Yes, of course, that seems reasonable.

**Comment by taylorludemann** [comment]
I would also like to create a button that would take the user to another page such as the homepage or a "start" page. Is that something we can create using a view? Is it built into the software?

**Comment by icc** [comment]
@stephensiano This is a bit off-topic. However, I believe you're going to have to create a custom JS to the page/node where you want this to display and then you use the xAPI events from the H5PContent to load different content once the user is done. 
There are some basic examples on [h5p.org](https://h5p.org/documentation/x-api) on how to listen for the events.

The Image Pairing content type is actually maintained on [Gitlab ](https://gitlab.com/icfoss/H5P-ImagePair)by another developer.

**Comment by stephensiano** [comment]
@icc I've added the post to https://github.com/jithin-space/h5p-image-pair/issues/11 I could really use some help! Thank you
